src/mame/snk/hng64.cpp

```diff
@@ -1720,20 +1720,48 @@ static const gfx_layout hng64_16x16x8_spritelayout =
 };
 
 static const uint32_t texlayout_xoffset[1024] = { STEP1024(0,8) };
-static const uint32_t texlayout_yoffset[512] = { STEP512(0,8192) };
-static const gfx_layout hng64_texlayout =
+static const uint32_t texlayout_yoffset[1024] = { STEP1024(0,8192) };
+
+static const uint32_t texlayout_xoffset_4[1024] = {
```

Comment:
Can this be precalculated?

Reply:
I'm not sure, it's just STEP1024(0,4) but with ^4 applied to get the pixels in the right order, I don't know if there's a macro for that.

Reply:
I mean something like init_texlayout_xoffset_4() in the driver class constructor.

Reply:
that appears to work, yes